databroker/broker.py

```diff
@@ -614,18 +622,7 @@ def __call__(self, text_search=None, **kwargs):
 
         return Results(res, self, data_key)
 
-    def find_headers(self, **kwargs):
-        "This function is deprecated."
-        warnings.warn("Use .__call__() instead of .find_headers()", stacklevel=2)
-        return self(**kwargs)
-
-    def fetch_events(self, headers, fill=True):
-        "This function is deprecated."
-        warnings.warn("Use .get_events() instead.", stacklevel=2)
-        return self.get_events(headers, fill=fill)
-
-    def fill_event(self, event, inplace=True,
-                   handler_registry=None, handler_overrides=None):
+    def fill_event(self, event, inplace=True, handler_registry=None):
```

Comment:
maybe `fill_event(self, event, inplace=True, field=None, handler_registry=None)` 
if `field=None` fill everything else fill only that field?